docker/README.md

```diff
@@ -0,0 +1,79 @@
+## Running DC/OS UI in Docker
+
+These instructions will guide you through building and launching the DC/OS UI using docker.
+
+### Requirements
+1. [Install docker](https://www.docker.com/products/overview)
+2. Clone dcos-ui repo
+
+  ```sh
+  git clone git@github.com:dcos/dcos-ui.git && cd dcos-ui
+  ```
+
+### Build Step
+
+You'll only need to run the build once. If you change the path to your plugins repo, you'll need to build again.
+
+1. Start docker on your machine
+2. Build the dcos-ui docker image. Run this command inside the the cloned `dcos-ui` repo.
+
+  **Option 1**. Build without a plugins repo
+    ```sh
+    docker build \
+      -t dcos-ui-browse:latest \
+      -f docker/Dockerfile.browse .
+    ```
+
+  **Option 2**. Build with a plugins repo
+
+    If you have a repository of plugins to enhance DC/OS UI, you may configure this by passing a URL to the git repo. For example: `https://{AUTH_TOKEN}:x-oauth-basic@github.com/my-company/dcos-ui-plugins.git`, where `{AUTH_TOKEN}` would be your generated token from, e.g. https://github.com/settings/tokens.
```

Comment:
@MatApple I'm trying to make it as agnostic to GH has possible. But you have a good point, it could be more verbose. I'll make a few changes.